master/getting-started/kubernetes/installation/hosted/kubeadm/1.7/calico.yaml

```diff
@@ -171,17 +171,20 @@ spec:
     spec:
       hostNetwork: true
       tolerations:
-      # this taint is set by all kubelets running `--cloud-provider=external`
-      # so we should tolerate it to schedule the calico pods
-      - key: node.cloudprovider.kubernetes.io/uninitialized
-        value: "true"
-        effect: NoSchedule
-      - key: node-role.kubernetes.io/master
-        effect: NoSchedule
-      # Allow this pod to be rescheduled while the node is in "critical add-ons only" mode.
-      # This, along with the annotation above marks this pod as a critical add-on.
-      - key: CriticalAddonsOnly
-        operator: Exists
+        # this taint is set by all kubelets running `--cloud-provider=external`
+        # so we should tolerate it to schedule the calico pods
+        - key: node.cloudprovider.kubernetes.io/uninitialized
```

Comment:
I don't think the `key: node.cloudprovider.kubernetes.io/uninitialized` one is needed anymore with the one below with no key specified. Right?